readme.md

```diff
@@ -23,3 +23,18 @@
 5) Run your tests
 
 To enable debugging, set `BUILDKITE_ANALYTICS_DEBUG_ENABLED=true`
+
+## Jest Collector Roadmap
+
+### DONE
+
+- [x] Baseline metric collection, such as test duration and pass/fail states
+- [x] Error messaging
```

Comment:
```suggestion
- [x] Send test pass/fail status on test completion
- [x] Send failure reason
```

Reply:
I lost your edits when I quickly in-browser resolved some conflicts yesterday @toolmantim . Have a look at my latest commit, should have them back now.